java/src/jmri/jmrit/operations/locations/schedules/tools/Bundle.java

```diff
@@ -0,0 +1,99 @@
+package jmri.jmrit.operations.locations.schedules.tools;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.util.Locale;
+import javax.annotation.CheckReturnValue;
+import javax.annotation.CheckForNull;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+@ParametersAreNonnullByDefault
+@CheckReturnValue
+@SuppressFBWarnings(value = "NM_SAME_SIMPLE_NAME_AS_SUPERCLASS", justification = "Desired pattern is repeated class names with package-level access to members")
+
+@javax.annotation.concurrent.Immutable
+
+/**
+ * Provides standard access for resource bundles in a package.
+ *
+ * Convention is to provide a subclass of this name in each package, working off
+ * the local resource bundle name.
+ *
+ * @author Bob Jacobsen Copyright (C) 2012
+ * @since 3.3.1
+ */
+public class Bundle extends jmri.jmrit.operations.Bundle {
+
+    @CheckForNull
+    private static final String name = "jmri.jmrit.operations.locations.JmritOperationsLocationsBundle"; // NOI18N
```

Comment:
No need to change. Just checking that you don't have a Bundle in package jmri.jmrit.operations.locations.schedules.